packages/contracts-bedrock/test/L1/OptimismPortal2.t.sol

```diff
@@ -439,6 +439,12 @@ contract OptimismPortal2_FinalizeWithdrawal_Test is CommonTest {
     constructor() {
         super.setUp();
 
+        // zero out contracts that should not be used
+        assembly {
```

Comment:
Honestly I think we can leave this out for now, if we're going to do stuff like this I would do it in a different PR after opening an issue to discuss the reasoning for it